installer/templates/phx_new/mix.exs

```diff
@@ -0,0 +1,58 @@
+defmodule <%= application_module %>.Mixfile do
+  use Mix.Project
+
+  def project do
+    [app: :<%= application_name %>,
+     version: "0.0.1",<%= if in_umbrella do %>
+     build_path: "../../_build",
+     config_path: "../../config/config.exs",
+     deps_path: "../../deps",
+     lockfile: "../../mix.lock",<% end %>
+     elixir: "~> 1.2",
```

Comment:
This file may need updating with the changes from https://github.com/phoenixframework/phoenix/commit/269c75ec8d1082915ce481f7b944a2bcc5c9cbe5